app/src/main/java/ar/com/wolox/android/cookbook/googleLogin/GoogleHelper.kt

```diff
@@ -0,0 +1,99 @@
+package ar.com.wolox.android.cookbook.googleLogin
+
+import android.content.Intent
+import android.support.v4.app.Fragment
+import android.view.View
+import ar.com.wolox.android.cookbook.googleLogin.model.UserGoogle
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.common.api.ApiException
+
+/**
+ * Set google login action to the view
+ *
+ * @param fragment
+ *  the fragment where the button is used
+ *
+ * @param resultCode
+ *  the code to catch the activity result
+ */
+fun View.setGoogleLoginAction(fragment: Fragment, resultCode: Int): UserGoogle? {
```

Comment:
Mmm, by doing this we are adding this function as a suggestion fo **ALL* `View`s in the project. We should find a solution with a smaller scope.

EDIT: I've taken a look at the code that uses this and we can move this to the `Fragment` that uses it. I see no value in extracting this since it's only used inside of it.